documentation/src/main/docs/concepts/model.md

```diff
@@ -72,6 +72,12 @@ Creating `Messages` is done using the {{ javadoc('io.eclipse.microprofile.reacti
 {{ insert('messages/MessageExamples.java', 'creation') }}
 ```
 
+Messages accept `null` as payload.
```

Comment:
I would warn the user that reactive streams do not support `null`, so you cannot consume a `Multi<P>` where the message (Message<P>) has a `null` payload.